docs/upgrade/UPGRADE-v8.0.0.md

```diff
@@ -91,6 +91,11 @@ There you can find links to upgrade notes for other versions too.
     - the namespace of `FriendlyUrlDataProviderInterface` and `FriendlyUrlDataProviderRegistry` has changed from `Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\CompilerPass` to `Shopsys\FrameworkBundle\Component\Router\FriendlyUrl` so change all your usages accordingly
     - you no longer need to tag your `FriendlyUrlDataProviders` with `shopsys.friendly_url_provider` because it is now done automatically
     - remove the usages of `RegisterFriendlyUrlDataProviderCompilerPass` class and `FriendlyUrlDataProviderRegistry::registerFriendlyUrlDataProvider` method because they have been removed
+- update your way of registration of `BreadcrumbGenerator` classes ([#1141](https://github.com/shopsys/shopsys/pull/1140))
+	- remove the usages of `FrontBreadcrumbResolverFactory` class as it has been removed.
+	- you no longer need to tag your `BreadcrumbGenerator` classes with `shopsys.breadcrumb_generator` tag as it is now done automatically
```

Comment:
The `shopsys.breadcrumb_generator` tag wasn't used before. Previous implementation relied on `FrontBreadcrumbResolverFactory` with hardcoded typed parameters that were autowired.

Reply:
Done